tests/integration/compose/common_test.go

```diff
@@ -17,8 +17,10 @@ const (
 	// Override FIFTYONE_DATABASE_ADMIN to true (until we can override via environment variable)
 	overrideFile = "../../tests/fixtures/docker/compose.override.yaml"
 	// To run the containers on macOS arm64, we need to set the platform
-	darwinOverrideFile = "../../tests/fixtures/docker/compose.override.darwin.yaml"
-	mongodbComposeFile = "../../tests/fixtures/docker/compose.override.mongodb.yaml"
+	darwinOverrideFile        = "../../tests/fixtures/docker/compose.override.darwin.yaml"
+	darwinOverrideFilePlugins = "../../tests/fixtures/docker/compose.override.darwin_plugins.yaml"
+	mongodbComposeFile        = "../../tests/fixtures/docker/compose.override.mongodb.yaml"
+	mongodbComposeFilePlugins = "../../tests/fixtures/docker/compose.override.mongodb_plugins.yaml"
```

Comment:
We need to override the teams-plugin image only when it is used. It's only used for our dedicated plugin test cases.  So I added additional override files.  (If added it to the base ones, it would create a plugin container - which we don't want).